shared/helpers/assertRole.js

```diff
@@ -1,4 +1,5 @@
-module.exports = web3 => async (acl, app, manager, appName, roleName, grantee = manager) => {
+module.exports = web3 => async (acl, app, manager, roleName, grantee = manager) => {
```

Comment:
Won't this have the problem that we can't differentiate two instances of the same app?

Reply:
Besides, it would be nice to move Multisig and Democracy to use this instead of `checkRole`.

Reply:
Ok, I see now this has been done in other PRs, forget it then :wink: